### ISSUE TITLE
Feature Request: AutoExportPolicy for Lustre

### ISSUE THREAD

**Issue by miikkahoo** [aws/aws-parallelcluster#5406]
When creating an Fsx for Lustre, there is no option to configure an auto export policy. It needs to be done when the association with S3 is done, it cannot be added afterwards.
I have an use case where output files would need to be downloaded from AWS for post-processing. Currently Export from Lustre to S3 needs to be done manually which could be automated.

Something like this would be awesome:
```yaml
SharedStorage:
  - StorageType: FsxLustre
  - FsxLustreSettings:
    AutoImportPolicy: NEW_CHANGED_DELETED
    AutoExportPolicy: NEW_CHANGED
```

**Comment by chenwany** [comment]
Hi!
The supported parameter for Fsx Lustre configuration in ParallelCluster comes from cloudformation template support. 
In the cloudformation documentation, it looks like there's no parameter related to automatically export updates to S3 bucket supported.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-filesystem-lustreconfiguration.html

**Comment by simons-orw** [comment]
Cloudformation does allow this to be configured through a [Data Repository Association](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-fsx-datarepositoryassociation.html) - 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-datarepositoryassociation-autoexportpolicy.html

**Comment by enrico-usai** [comment]
Added with https://github.com/aws/aws-parallelcluster/pull/5693 and released as part of 3.8.0
